### PR TITLE
Add filter for accept fund on gnosis

### PR DESCRIPTION
**Pull request by mohammadranjbarz**
related to #664

### CODE REVIEW

File: src/resolvers/projectResolver.test.ts [hunk 1]
```diff
@@ -510,6 +512,137 @@ function projectsTestCases() {
...
+  //     ),
+  //   );
+  //   assert.isOk(
+  //     project.addresses.find(
+  //       address =>
+  //         address.isRecipient === true &&
+  //         address.networkId === NETWORK_IDS.MAIN_NET,
+  //     ),
+  //   );
+  // });
```
Comment (mohammadranjbarz) [inline src/resolvers/projectResolver.test.ts]: @CarlosQ96  I couldn't find a way to pass this test but as I explained in comment, the frontend doesn't use that, so it's safe , but if you can improve the query to pass this, I would appreciate it

File: src/server/bootstrap.ts [hunk 2]
```diff
@@ -268,18 +265,6 @@ export async function bootstrap() {
...
-    ).getAuthUrl({
-      // trackId: generateRandomEtheriumAddress(),
-      trackId: 'STATE',
-    });
-    logger.info('twitter auth url', authUrl);
-    // const accessToken = await twitterAdapter.getUserInfoByOauth2Code({
-    //   oauth2Code:
-    //     'SDg1b1otX3lYaURDZjN3emQtTjVwVUMwMFNmeGFjQ0tLWlBNQnhobEszQ19hOjE2NTcxMTU5MjIzMjg6MTowOmFjOjE',
-    // });
-    // logger.info('twitter accessToken', accessToken);
```
Comment (mohammadranjbarz) [inline src/server/bootstrap.ts:282]: There was an unnecassary and unused code

File: src/resolvers/projectResolver.ts [hunk 3]
```diff
@@ -441,6 +443,12 @@ export class ProjectResolver {
       return query.andWhere(`project.${filter} ${isRequested} NULL`);
     }
 
+    if (filter === 'acceptFundOnGnosis' && filterValue) {
+      return query.andWhere(
+        `"isRecipient"=true AND "networkId"=${NETWORK_IDS.XDAI}`,
+      );
```
Comment (mohammadranjbarz) [inline src/resolvers/projectResolver.ts]: They will query on `addresses` entity ( the we joined it to project)

**Review by CarlosQ96 — APPROVED** [review]
LGTM from my side, all tests pass now. You can test the subquery, but everything else looks good!

**Pull request merged**